uploadNeuroDB/minc_insertion.pl

```diff
@@ -145,44 +145,44 @@
 &Getopt::Tabular::SetHelp($Help, $Usage);
 &Getopt::Tabular::GetOptions(\@opt_table, \@ARGV) || exit 1;
 
-
-# input option error checking
-{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
-
-
-if ($profile && !@Settings::db) { 
-    print "\n\tERROR: You don't have a ".
-    "configuration file named '$profile' in:  $ENV{LORIS_CONFIG}/.loris_mri/ \n\n";
-    exit 2; 
+if(!$ENV{LORIS_CONFIG}) {
+	print STDERR "\n\tERROR: Environment variable 'LORIS_CONFIG' not set\n\n";
+	exit 1;  # TODO replace by appropriate exit code from ExitCodes once it is defined
 }
 
-if (!$profile) { 
+if (!defined$profile || !-e "$ENV{LORIS_CONFIG}/.loris_mri/$profile") {
     print $Help; 
-    print "$Usage\n\tERROR: You must specify a valid ".
-    "and existing profile.\n\n";  
-    exit 3; 
+    print STDERR "$Usage\n\tERROR: You must specify a valid and existing profile.\n\n";  
+    exit 2; #TODO 1: use exit code from ExitCodes once it is merged
 }
 
 if ( !($tarchive xor $upload_id) ) {
-    print "\nERROR: You should either specify an upload ID or a tarchive ".
-        "path. Make sure that you set only one of those options. ".
-        "Upload will exit now.\n\n\n";
-    exit; #TODO 1: use exit code from ExitCodes once it is merged
+    print STDERR "\nERROR: You should either specify an upload ID or a tarchive "
+        . "path. Make sure that you set only one of those options. "
+        . "Upload will exit now.\n\n\n";
+    exit 3; #TODO 1: use exit code from ExitCodes once it is merged
 }
 
-if ($tarchive && !(-e $tarchive) ) {
-    print "\nERROR: Could not find archive $tarchive. \nPlease, make sure ".
-          " the path to the archive is correct. Upload will exit now.\n\n\n";
-    exit 4;
+if (defined $tarchive && !(-e $tarchive) ) {
+    print STDERR "\nERROR: Could not find archive $tarchive. \nPlease, make sure "
+        . " the path to the archive is correct. Upload will exit now.\n\n\n";
+    exit 3;#TODO 1: use exit code from ExitCodes once it is merged
 }
-unless (-e $minc) {
-    print "\nERROR: Could not find minc $minc. \nPlease, make sure the ".
-          "path to the minc is correct. Upload will exit now.\n\n\n";
-    exit 5;
+if (!defined $minc || !-e $minc) {
+    print STDERR "$Usage\n\tERROR: You must specify a valid and existing "
+        . "MINC file with -minc.\n\n";  
+    exit 3;#TODO 1: use exit code from ExitCodes once it is merged
 }
 
+# input option error checking
+{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
 
-
+if (!@Settings::db) { 
+    printf STDERR
+        "\n\tERROR: no database setting '\@db' found in %s\n",   
+        "$ENV{LORIS_CONFIG}/.loris_mri/$profile or file unreadable";
+    exit 4; #TODO 1: use exit code from ExitCodes once it is merged
```

Comment:
are theswe changes all going to conflict with the Exit code harmonization?!

Reply:
See my comment on exit codes